lidar_lite_v3hp/launch/lidar_litev3hp.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+  <node pkg="rosserial_python" type="serial_node.py" name="serial_node1">
+    <param name="port" value="/dev/ttyACM0"/>
+    <param name="baud" value="57600"/>
+  </node>
+  <node pkg="rosserial_python" type="serial_node.py" name="serial_node2">
+    <param name="port" value="/dev/ttyACM1"/>
+    <param name="baud" value="57600"/>
+  </node>
+  <param name="/distance_between_two_sensor" type="double" value="2.0" />
```

Comment:
This parameter should be private to the lidar_litev3hp node. Additionally, since it is a calibration parameter you should read it in from a yaml file located in a directory provided as a roslaunch argument. We'll probably need to discuss so you can understand what I mean.